app/Repositories/Eloquents/PermissionEloquentRepository.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: quantien
+ * Date: 11/8/18
+ * Time: 8:52 AM
+ */
+
+namespace App\Repositories\Eloquents;
+
+
+use App\Models\Permission;
+use App\Repositories\Contracts\PermissionRepository;
+
+class PermissionEloquentRepository extends AbstractEloquentRepository implements PermissionRepository
+{
+
+    public function model()
+    {
+        // TODO: Implement model() method.
+        return new Permission;
+    }
+
+    public function getAll($data = [])
+    {
+        // TODO: Implement getAll() method.
+        return $this->model()->all();
+    }
+
+    public function store($data = [])
+    {
+        // TODO: Implement store() method.
+        return $this->model()->create($data);
+    }
+
+    public function find($id)
+    {
+        // TODO: Implement find() method.
+        return $this->model()->findOrFail($id);
+    }
+
+    public function update($id, $data = [])
+    {
+        // TODO: Implement update() method.
+        $model = $this->model()->findOrFail($id);
+
+        return $model->update($data);
+    }
+
+    public function show($id)
+    {
+        // TODO: Implement show() method.
+        return $this->model()->findOrFail($id);
+    }
+
+    public function delete($id)
+    {
+        // TODO: Implement delete() method.
+        $model = $this->model()->findOrFail($id);
+
+        return $model->delete();
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PermissionEloquentRepository.php</br>**Line**: 63</br>Expected 1 newline at end of file; 0 found